.github/workflows/production.yml

```diff
@@ -1,4 +1,5 @@
 name: Production
+concurrency: firebase_deploy
```

Comment:
could you carry some comments from the PR into this file as well for documentation? Thanks!